data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -48,7 +77,10 @@ def get_bq_json_for_survey_questions_response_json(
         "questions": [
             {
                 "question_id": question_response_json["id"],
-                "question_title": question_response_json["headings"][0]["heading"]
+                "question_title": question_response_json["headings"][0]["heading"],
+                "question_type": question_response_json["family"],
+                "question_subtype": question_response_json["subtype"],
+                "question_answers": [parse_answers_in_question_json(question_response_json)]
```

Comment:
I guess we don't need to wrap it into an array? It seems we are using an array, which itself contains multiple arrays.

Reply:
Will try it to see the result.